src/routes/components/components.json

```
@@ -1,4 +1,16 @@
 [
+	{
+	    "title": "svelte-lazy-loader",
+	    "url": "https://github.com/sawyerclick/svelte-lazy-loader",
+	    "description": "A lazy-loader component library for Svelte",
+	    "npm": "svelte-lazy-loader",
+	    "addedOn": "2022-02-27",
+	    "tags": [
+		"images",
+		"components and libraries"
+	    ],
+	    "stars": 0
+	}
```

There is a missing comma here
```suggestion
	},
```